src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -823,44 +823,16 @@ public Task ExecuteCommand(PSCommand psCommand)
 
         internal static TResult ExecuteScriptAndGetItem<TResult>(string scriptToExecute, Runspace runspace, TResult defaultValue = default(TResult))
         {
-            Pipeline pipeline = null;
-
-            try
+            using (var pwsh = PowerShell.Create())
```

Comment:
Same comment on `var`

Reply:
I left it as is this time since it's the pattern we currently use everywhere.